src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2004.OptimizeFileSize_Valid.sarif

```diff
@@ -42,6 +42,25 @@
               }
             }
           ]
+        },
+        {
+          "ruleIndex": -1,
+          "rule": {
```

Comment:
Don't specify `ruleIndex` here. The default is -1, so if it it missing, it will be set to -1 when we deserialize the log file.